doc/installation.rst

```diff
@@ -166,6 +166,14 @@ macOS versions:
       - :package-release-tgz:`TAR <macos_big_sur>`
       - :package-dev-tgz:`TAR <macos_big_sur>`
 
+    * - Monterey (12)
```

Comment:
Should we remove the dev entries for Catalina and Big Sur then?

Reply:
Good catch, I'll remove them.